### PR TITLE
Fixes to documentation after repository migration

### PR DESCRIPTION
**Pull request by hlabas**
Proposing to add a set of common requirements and steps at the root of all samples to simplify each individual sample documentation.

### CODE REVIEW

File: samples/json-editor/README.md [hunk 1]
```diff
@@ -6,33 +6,33 @@ This extension provides a JSON formatter and validator based on the [Codemirror]
...
 ```
-
-### Upload extension
-If you want to inline all dependencies and upload the extension entirely to Contentful:
-```bash
-contentful-extension update --descriptor extension.debug.json --space-id <yourSpaceId> --force
-```
-
-Please note that as soon as the concatenated file gets over 200k, you'll have to host it on your own server, using HTTPS and with CORS enabled.
+The [same constraints](../README.md) relative to loading unsafe scripts apply.
```
Comment (ChrisChinchilla) [inline samples/json-editor/README.md]: @hlabas I don't see any mention of these constraints

**Comment by ChrisChinchilla** [comment]
@geigerzaehler @hlabas I made some changes, just waiting for response on my comments regarding:

```
The [same constraints](../README.md) relative to loading unsafe scripts apply.
```

As I'm not sure what it's referring to.

**Comment by hlabas** [comment]
Ok, this should be reworded since it was obviously unclear, but it refers to the notes related to debugging from your local environment, mentioned line 30 of the samples/README.md :
### Notes relative to debugging from your local environment

**Comment by ChrisChinchilla** [comment]
OK, done @hlabas I think this repo will need further refinement over time, but this addresses what you needed done.

**Pull request merged**